solidity/ecdsa/contracts/test/TStub.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.6;
+
+import "@threshold-network/solidity-contracts/contracts/token/T.sol";
+
+// This contract is added just to get artifacts and typings generated for T.
```

Comment:
We can also use hardhat dependency compiler plugin, see [here](https://github.com/keep-network/coverage-pools/blob/31d3af2039adc8a556e2ed125b28af4a7a16a371/hardhat.config.ts#L24-L27) as well as this PR https://github.com/keep-network/coverage-pools/pull/206 for a full example (`.gitignore`, `.prettierignore`, `.solhintignore`).

Reply:
Thanks! aeb60b7c1ad463c21fbb28a5d6e96ce20998d985